packages/perseus-editor/src/components/interactive-graph-settings.tsx

```diff
@@ -117,8 +149,9 @@ class InteractiveGraphSettings extends React.Component<Props, State> {
     labelXRef = React.createRef<HTMLInputElement>();
     labelYRef = React.createRef<HTMLInputElement>();
 
-    static stateFromProps(props) {
+    static stateFromProps(props: Props, state?: State) {
         return {
+            isExpanded: state?.isExpanded ?? true,
```

Comment:
I did this so we'd retain the expanded state even in the case where the props change enough to re-build state. See `UNSAFE_componentWillReceiveProps`